lib/edda-server/src/change_set_processor_task/materialized_view.rs

```diff
@@ -486,6 +543,7 @@ fn mv_dependency_graph() -> Result<DependencyGraph<ReferenceKind>, MaterializedV
     add_reference_dependencies_to_dependency_graph!(dependency_graph, ActionViewListMv);
     add_reference_dependencies_to_dependency_graph!(dependency_graph, ActionPrototypeViewListMv);
     add_reference_dependencies_to_dependency_graph!(dependency_graph, ComponentViewListMv);
+    add_reference_dependencies_to_dependency_graph!(dependency_graph, ComponentViewMv);
```

Comment:
Note to self: don't forget to add all new MVs! We should make this a compile-time check...